### PR TITLE
wt: add build dependencies

### PR DESCRIPTION
**Pull request by Danoloan10**
Fixes https://github.com/epoupon/lms/issues/384

#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glib)

### CODE REVIEW

**Comment by classabbyamp** [comment]
this isn't Gentoo, not everything needs to be a build option. some of these don't even make sense, like the qt4 one (we don't have qt4 anymore)

**Comment by Danoloan10** [comment]
the default config would fix the original issue, because everything is enabled by default on the condition that each dependency package is found. so I guess a full build without options should be ok. in that case should any of these options be kept?

**Comment by classabbyamp** [comment]
then just add the necessary build dependencies

File: srcpkgs/wt/template [hunk 1]
```diff
@@ -3,7 +3,12 @@ pkgname=wt
 version=4.10.0
 revision=2
 build_style=cmake
-makedepends="boost-devel"
+build_helper=qmake
+configure_args="-DBUILD_EXAMPLES=OFF -DINSTALL_EXAMPLES=OFF -DENABLE_QT4=OFF $(vopt_bool firebird ENABLE_FIREBIRD)"
```
Comment (classabbyamp) [inline srcpkgs/wt/template:7]: INSTALL_EXAMPLES appears to default to off

File: srcpkgs/wt/template [hunk 2]
```diff
@@ -3,7 +3,12 @@ pkgname=wt
 version=4.10.0
 revision=2
```
Comment (classabbyamp) [inline srcpkgs/wt/template:4]: this should be incremented

File: srcpkgs/wt/template [hunk 3]
```diff
@@ -3,7 +3,12 @@ pkgname=wt
...
 build_style=cmake
-makedepends="boost-devel"
+build_helper=qmake
+configure_args="-DBUILD_EXAMPLES=OFF -DINSTALL_EXAMPLES=OFF -DENABLE_QT4=OFF $(vopt_bool firebird ENABLE_FIREBIRD)"
+hostmakedepends="qt5-host-tools qt5-qmake"
+makedepends="boost-devel openssl-devel Haru-devel pango-devel sqlite-devel postgresql-libs-devel
+ libmysqlclient-devel unixodbc-devel qt5-devel libfreeglut-devel libunwind-devel
+ $(vopt_if firebird firebird3)"
```
Comment (classabbyamp) [inline srcpkgs/wt/template:11]: firebird probably needs to be added to `depends` (too?)

Comment (Danoloan10) [reply srcpkgs/wt/template:11]: I have investigated the code base a bit and it looks like it's looking for "libfbclient", which is not being provided by the firebird3 package anyway, so this option is useless

File: srcpkgs/wt/template [hunk 4]
```diff
@@ -1,9 +1,13 @@
 # Template file for 'wt'
 pkgname=wt
 version=4.10.0
-revision=2
+revision=3
 build_style=cmake
-makedepends="boost-devel"
+build_helper=qmake
+configure_args="-DBUILD_EXAMPLES=OFF -DENABLE_QT4=OFF $(vopt_bool firebird ENABLE_FIREBIRD)"
```
Comment (classabbyamp) [inline srcpkgs/wt/template:7]:
```suggestion
configure_args="-DBUILD_EXAMPLES=OFF -DENABLE_QT4=OFF"
```

**Pull request merged**